### DISCUSSION TITLE
How to use IPU in autogluon?

### DISCUSSION THREAD

**Discussion by muazhari** [autogluon/autogluon#2457] [Q&A]
I'm using the Paperspace machine to train the model. However, when I run the fit method, Autogluon only recognizes the IPU and not using it. How to make Autogluon using the available IPU?

**Comment by callumm-graphcore** [comment]
Hello,

I assume you're the same person who asked about this on StackOverflow, but for anyone else who might see this I'll repost the relevant part of my answer from there here:

> As far as I can tell, the Autogluon library does not currently support using IPUs. The Poplar SDK supports PyTorch and PyTorch Lightning, which Autogluon is based on, so the library could in principle be supported. I'd be really interested to hear more about what you want to use Autogluon for!
> 
> In the meantime, there are many IPU resources available, including:
> 
> - Public examples, including a wide range of applications written for the IPU: https://github.com/graphcore/examples
> - Tutorials: https://github.com/graphcore/tutorials
> - The Graphcore Optimum library, which has optimised implementations of HuggingFace transformer models: https://github.com/huggingface/optimum-graphcore
> - The model garden, with links to the public examples and Paperspace notebooks: https://www.graphcore.ai/resources/model-garden
> 
> I'm sorry that what you want isn't supported, but I hope that the above is at least helpful.

With thanks,

Callum
